data/ devices/ samsung/02_galaxy_a_j.yml

```diff
@@ -549,3 +549,42 @@
     default_brightness: 16200
     pixel_format: ABGR_8888
     theme: portrait_hdpi
+---
+
+- name: Samsung Galaxy A7 (2017) (Exynos)
+  id: a7y17lte
+  codenames:
+      - SM-A720F
+      - a7y17lte
+  architecture: 
```

Comment:
The `architecture` field should be `armeabi-v7a` (if 32-bit) or `arm64-v8a` (if 64-bit).